src/components/DateRangePickerInput.jsx

```diff
@@ -41,6 +41,7 @@ const propTypes = forbidExtraProps({
   isEndDateFocused: PropTypes.bool,
   showClearDates: PropTypes.bool,
   disabled: PropTypes.bool,
+  selectivelyDisabled: PropTypes.string,
```

Comment:
Generally we've been moving in the direction, for > 2-state props, of having a single enum prop instead of multiple boolean props. (With your suggestion, we'd want to specify a `mutuallyExclusiveTrueProp` propType to prevent both from being true anyways)